docs/flowlayout.md

```diff
@@ -18,6 +23,110 @@ FlowColumn {
 }
 ```
 
+## Migration Guide to the official FlowLayouts
+
+1. Replace import packages to point to Androidx.Compose
+``` kotlin
+import androidx.compose.foundation.layout.FlowColumn
+```
+
+``` kotlin
+import androidx.compose.foundation.layout.FlowRow
+```
+
+For `FlowColumn`:
+2. Replace Modifier `mainAxisAlignment` with `verticalArrangement`
+3. Replace Modifier `crossAxisAlignment` with `horizontalAlignment`
+
+For `FlowRow`
+4. `mainAxisAlignment` is now `horizontalArrangement`
+5. `crossAxisAlignment` is now `verticalAlignment`
+
+``` kotlin
+FlowColumn(
+    modifier = Modifier,
+    verticalArrangement = Arrangement.Top,
+    horizontalAlignment: = Alignment.Start,
+    content = { // columns }
+) 
+```
+
+``` kotlin
+FlowRpw(
+    modifier = Modifier,
+    horizontalArrangement = Arrangement.Start,
+    verticalAlignment: = Alignment.Top,
+    content = { // rows }
+) 
+```
+
+6. Replace `mainAxisSpacing` with `VerticalArrangement.spacedBy(50.dp)` in `FlowColumn` and `HorizontalArrangement.spacedBy(50.dp)` in `FlowRow`
+``` kotlin
+FlowColumn(
+    verticalArrangement = VerticalArrangement.spacedBy(50.dp),
+    content = { // columns }
+)
+```
+
+``` kotlin
+FlowRow(
+    horizontalArrangement = HorizontalArrangement.spacedBy(50.dp),
+    content = { // rows }
+)
+```
+
+7. `crossAxisSpacing` can be supported by adding a padding to each child
+
+``` kotlin
+FlowRow(
+    horizontalArrangement = HorizontalArrangement.spacedBy(50.dp),
+    { Box(Modifier.padding(bottom = 20.dp) }
```

Comment:
This will add a padding to each child in the FlowRow, as each box will have this padding and separate themselves from the next row. 

``` kotlin
FlowRow(
    { repeat(10) { Box(Modifier.padding(bottom = 20.dp) } }
)
```

We decided not to support crossAxisArrangement in this iteration, based on our design conversations, as we felt there might not be a strong need for this. However, we do support spacing in the basic level by adding spacing to the children, much like you would padding to a text within a box.

Reply:
Could we document the difference in the migration guide here, since replacing `crossAxisSpacing` with padding on the children isn't a 1:1 replacement? I also created https://issuetracker.google.com/issues/268365538 to track that.